maintenance/wikia/cronjobs/updateFandomShopCache.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+/**
+ * Get data from intentX API and update memcache 
+ *
+ * @group cronjobs
+ * @see update-fandom-shop-cache.yaml
+ *
+ */
+
+require_once __DIR__ . '/../../Maintenance.php';
+
+class updateFandomShopCache extends Maintenance {
+
+    public function execute() {
+        global $wgFandomShopMap, $wgFandomShopMapDev;
+
+        echo 'Running maintenence script';
+
+        $logger = \Wikia\Logger\WikiaLogger::instance();
+        $logger->info( 'Updating Fandom Shop Cache' );
+
+        if ( Wikia::isDevEnv() ) {
+            $url = "https://community.chris.fandom-dev.us/wikia.php?controller=DesignSystemApi&method=getFandomShopDataFromIntentX&id=$key";
+            $shopMap = $wgFandomShopMapDev;
+        } else {
+            $url = "https://community.fandom.com/wikia.php?controller=DesignSystemApi&method=getFandomShopDataFromIntentX&id=$key";
+            $shopMap = $wgFandomShopMap;
+        }
+
+        foreach ( $shopMap as $key => $value) {
+            $handle = curl_init();
+            curl_setopt($handle, CURLOPT_URL, $url);
+            curl_setopt($handle, CURLOPT_RETURNTRANSFER, true);
+            $data = curl_exec($handle);
+            $allCommunitiesUpdated = true;
+
+            $responseCode   = curl_getinfo($handle, CURLINFO_HTTP_CODE);
+
+            if ( curl_errno($handle) ) {
+                print curl_error($handle);
+                $logger->error( "There has been an error updating $value shop cache" );
```

Comment:
Let's add the error message and error code to the log message context (2nd parameter).